qa/vagrant/src/test/resources/packaging/scripts/25_tar_plugins.bats

```diff
@@ -166,44 +147,40 @@ setup() {
     assert_file_exist "$TEMP_CONFIG_DIR/elasticsearch.yml"
 
     # Checks that plugin archive is available
-    [ -e "$SHIELD_ZIP" ]
+    [ -e "$EXAMPLE_PLUGIN_ZIP" ]
 
     # Install Shield with the CONF_DIR environment variable
-    run env "CONF_DIR=$TEMP_CONFIG_DIR" /tmp/elasticsearch/bin/plugin install "elasticsearch/shield/latest" -u "file://$SHIELD_ZIP"
+    run env "CONF_DIR=$TEMP_CONFIG_DIR" /tmp/elasticsearch/bin/plugin install jvm-example -u "file://$EXAMPLE_PLUGIN_ZIP"
     [ "$status" -eq 0 ]
 
     # Checks that Shield is correctly installed
-    assert_file_exist "/tmp/elasticsearch/bin/shield"
-    assert_file_exist "/tmp/elasticsearch/bin/shield/esusers"
-    assert_file_exist "/tmp/elasticsearch/bin/shield/syskeygen"
-    assert_file_exist "$TEMP_CONFIG_DIR/shield"
-    assert_file_exist "$TEMP_CONFIG_DIR/shield/role_mapping.yml"
-    assert_file_exist "$TEMP_CONFIG_DIR/shield/roles.yml"
-    assert_file_exist "$TEMP_CONFIG_DIR/shield/users"
-    assert_file_exist "$TEMP_CONFIG_DIR/shield/users_roles"
-    assert_file_exist "/tmp/elasticsearch/plugins/shield"
+    # TODO add bin to jvm-example
+    # assert_file_exist "/tmp/elasticsearch/bin/shield"
```

Comment:
I'll drop the shield references next. I have a patch for adding bin to
jvm-example and will use that to replace all those bin checks. I can do
that in this commit if I merge it first and rebase. I don't have a link
because I'm on mobile.
On Aug 14, 2015 2:41 PM, "Simon Willnauer" <notifications@github.com> wrote:

> In qa/vagrant/src/test/resources/packaging/scripts/25_tar_plugins.bats
> <https://github.com/elastic/elasticsearch/pull/12895#discussion_r37122263>
> :
>
> >      [ "$status" -eq 0 ]
> >
> >      # Checks that Shield is correctly installed
> > -    assert_file_exist "/tmp/elasticsearch/bin/shield"
> > -    assert_file_exist "/tmp/elasticsearch/bin/shield/esusers"
> > -    assert_file_exist "/tmp/elasticsearch/bin/shield/syskeygen"
> > -    assert_file_exist "$TEMP_CONFIG_DIR/shield"
> > -    assert_file_exist "$TEMP_CONFIG_DIR/shield/role_mapping.yml"
> > -    assert_file_exist "$TEMP_CONFIG_DIR/shield/roles.yml"
> > -    assert_file_exist "$TEMP_CONFIG_DIR/shield/users"
> > -    assert_file_exist "$TEMP_CONFIG_DIR/shield/users_roles"
> > -    assert_file_exist "/tmp/elasticsearch/plugins/shield"
> > +    # TODO add bin to jvm-example
> > +    # assert_file_exist "/tmp/elasticsearch/bin/shield"
>
> and here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elastic/elasticsearch/pull/12895/files#r37122263>.
>